<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog1</class>
 <widget class="QDialog" name="Dialog1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>427</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>656</width>
    <height>427</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>656</width>
    <height>427</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(170, 170, 255)</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>370</y>
     <width>251</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(191, 191, 191);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>160</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QCheckBox" name="m_spine">
      <property name="text">
       <string>Spine</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="m_hipC">
      <property name="text">
       <string>Hip Center</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="m_hipR">
      <property name="text">
       <string>Hip Right</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="m_hipL">
      <property name="text">
       <string>Hip Left</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="m_kneeR">
      <property name="text">
       <string>Knee Right</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="m_kneeL">
      <property name="text">
       <string>Knee Left</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="m_footR">
      <property name="text">
       <string>Foot Right</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="m_footL">
      <property name="text">
       <string>Foot Left</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>135</width>
     <height>332</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="m_head">
      <property name="text">
       <string>Head</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="m_shldC">
      <property name="text">
       <string>Shoulder Center</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="m_shldR">
      <property name="text">
       <string>Shoulder Right</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="m_shldL">
      <property name="text">
       <string>Shoulder Left</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="m_elbowR">
      <property name="text">
       <string>Elbow Right</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="m_elbowL">
      <property name="text">
       <string>Elbow Left</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="m_wristR">
      <property name="text">
       <string>Wrist Right</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="m_wristL">
      <property name="text">
       <string>Wrist Left</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>40</y>
     <width>251</width>
     <height>301</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Downloads/cuerpo.png</pixmap>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>20</width>
     <height>331</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="led1">
   <property name="geometry">
    <rect>
     <x>513</x>
     <y>74</y>
     <width>10</width>
     <height>10</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0%)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="led2">
   <property name="geometry">
    <rect>
     <x>513</x>
     <y>112</y>
     <width>10</width>
     <height>10</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0%)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="led3">
   <property name="geometry">
    <rect>
     <x>492</x>
     <y>112</y>
     <width>10</width>
     <height>10</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0%)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="led4">
   <property name="geometry">
    <rect>
     <x>534</x>
     <y>112</y>
     <width>10</width>
     <height>10</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0%)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="led5">
   <property name="geometry">
    <rect>
     <x>484</x>
     <y>160</y>
     <width>10</width>
     <height>10</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0%)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="led6">
   <property name="geometry">
    <rect>
     <x>541</x>
     <y>160</y>
     <width>10</width>
     <height>10</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0%)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="led7">
   <property name="geometry">
    <rect>
     <x>478</x>
     <y>188</y>
     <width>10</width>
     <height>10</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0%)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="led8">
   <property name="geometry">
    <rect>
     <x>548</x>
     <y>188</y>
     <width>10</width>
     <height>10</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0%)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="led9">
   <property name="geometry">
    <rect>
     <x>513</x>
     <y>144</y>
     <width>10</width>
     <height>10</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0%)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="led10">
   <property name="geometry">
    <rect>
     <x>513</x>
     <y>188</y>
     <width>10</width>
     <height>10</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0%)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="led11">
   <property name="geometry">
    <rect>
     <x>499</x>
     <y>188</y>
     <width>10</width>
     <height>10</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0%)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="led12">
   <property name="geometry">
    <rect>
     <x>527</x>
     <y>188</y>
     <width>10</width>
     <height>10</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0%)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="led13">
   <property name="geometry">
    <rect>
     <x>501</x>
     <y>243</y>
     <width>10</width>
     <height>10</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0%)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="led14">
   <property name="geometry">
    <rect>
     <x>525</x>
     <y>243</y>
     <width>10</width>
     <height>10</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0%)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="led15">
   <property name="geometry">
    <rect>
     <x>502</x>
     <y>287</y>
     <width>10</width>
     <height>10</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0%)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="led16">
   <property name="geometry">
    <rect>
     <x>525</x>
     <y>287</y>
     <width>10</width>
     <height>10</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0%)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="all">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>370</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Select all</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog1</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog1</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
